masterlist.yaml

```
@@ -17407,6 +17452,28 @@ plugins:
       - crc: 0xDED3A723
         util: 'SSEEdit v4.0.2f'
 
+  - name: 'Sunthgat.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/55343/' ]
+    dirty:
+      - <<: *reqManualFix
+        crc: 0x7F7C712F
+        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        nav: 1
+  
+  - name: 'Syerscote.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/58403/' ]
+    dirty:
+      - <<: *reqManualFix
+        crc: 0x80E9D9C0
+        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 4
+        udr: 4
+        nav: 2
+      - <<: *reqManualFix
+        crc: 0xFD079DBD
+        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        nav: 2
```

It looks like the latest version of this mod, v2.1, has different cleaning data:

```suggestion
      - <<: *quickClean
        crc: 0xF79DBD87
        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        udr: 4
```